---
title: Research
nav:
  order: 1
  tooltip: Main Research Lines
---

# {% include icon.html icon="fa-solid fa-microscope" %}Research

##  3D CHROMATIN ORGANIZATION

To trigger gene expression, regulatory elements are physically brought into the vicinity of promoters in a process called “looping”. Although the process of gene transcription has been largely studied, the principles of how regulatory elements engage in loops and find their appropriate partner, often ignoring other nearby genes, are still largely unknown.

The study of the 3D organization of genomes through Chromosome Conformation Capture (3C) and derivatives (for a review, see de [de Wit and de Laat, 2012](https://www.ncbi.nlm.nih.gov/pubmed/22215806)) revealed that these enhancer-promoter associations are usually confined within Topologically Associating Domains (TADs). These megabase-sized domains represent broad DNA regions containing loci that interact more frequently with themselves than with the rest of the genome ([Dixon et al., 2012](https://www.ncbi.nlm.nih.gov/pubmed/22495300); [Nora et al., 2012](https://www.ncbi.nlm.nih.gov/pubmed/22495304)). TADs display a high degree of overlap with previously described regulatory landscapes where enhancers are able to exert their influence. Based on this, it was speculated that TADs represent fundamental genomic modules that facilitate regulatory elements to find their cognate promoters. 

We were among the first to demonstrate that TADs are biologically relevant: the disruption of their boundaries may cause these structures to intermingle, causing novel associations between otherwise segregated enhancer and promoter pairs ([Lupiáñez et al., 2015](https://www.ncbi.nlm.nih.gov/pubmed/25959774); [Bianco et al., 2018](https://pubmed.ncbi.nlm.nih.gov/29662163/)). Such interactions can result in aberrant patterns of gene expression and induce congenital malformations or cancer ([Lupiáñez et al., 2016](https://www.ncbi.nlm.nih.gov/pubmed/26862051); [Anania et al., 2020](https://pubmed.ncbi.nlm.nih.gov/29692413/)). Moreover, we recently showed that changes in TAD organization can also serve as a substrate for the evolution of novel phenotypes, such as the intersexuality of female moles ([Real et al., 2020](https://pubmed.ncbi.nlm.nih.gov/33033216/)) or the enlarged fins of skates ([Marlétaz et al., 2023](https://pubmed.ncbi.nlm.nih.gov/37046085/)).

In ongoing projects, we seek to decipher developmental principles of 3D chromatin organization ([Anania et al., 2022](https://pubmed.ncbi.nlm.nih.gov/35817979/)) and to understand how they may have evolved ([Acemel et al., 2023](https://pubmed.ncbi.nlm.nih.gov/36603519/)).

{% include section.html %}

## SEX DETERMINATION

In vertebrates, both testes and ovaries derive from a common precursor organ: the bipotential gonad.

In mammals, sex determination is triggered by the early expression of either SRY in the bipotential XY gonad, or its absence in the XX counterpart (for a review, see [Capel, 2017](https://www.ncbi.nlm.nih.gov/pubmed/28804140)). Upon commitment of the gonad to a specific fate, a complex genetic and hormonal cascade induces alterations on cell identity, on tissue structure, and, ultimately, a series of anatomical and behavioral changes in the entire organism.

Although the outcome of sex determination is conserved across species, the mechanisms of sex determination are astonishingly plastic. There is not an initiator of sex differentiation that is universal for all species. Input signals are rather variable and controlled by genetic (GSD) or environmental mechanisms (ESD), or by a combination of both. The evolutionary advantages that result from such a plastic system are still uncertain, but could be related to a certain degree of phenotypical variance within male and female categories, which may be highly adaptive in changing environments, as long as there are two compatible sexes.

In ongoing projects, we aim to understand the molecular sources of the evolutionary plasticity of sex determination. For this purpose, we are profiling and comparing the process of sex determination in a wide range of vertebrate species to identify sources of genomic and regulatory variation. We are particularly interested in evaluating the potential role of 3D chromatin organization ([Mota-Gómez et al., 2022](https://www.biorxiv.org/content/10.1101/2022.11.18.516861v1)) and identifying novel regulators of sex determination ([Hurtado et al., 2023](https://www.nature.com/articles/s41467-024-47658-x)).

{% include section.html %}

##  METHODS
Here you can know about the relevant methodology of the lab:

- Single-cell omics (scRNA-seq, scATAC-seq)
- <details>
  <summary>Single-cell omics</summary>
  Here is more information that appears when you click.
</details>
- Quantification of chromatin interactions (4C-seq, Hi-C, capture-Hi-C, promoter capture-C)
- Identification of regulatory elements (ChiP-seq, ATAC-seq)
- Genome editing (CRISPR/Cas on mouse ES cells)
- Generation of transgenic animals (tetraploid complementation assay)

